src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -612,6 +617,25 @@ public AdsBaseObject triggerAdSlot(String slotName) {
     return this;
   }
 
+  private void triggerBLB() {
+    scrollToFooter();
+
+    simulateUserActivity(Duration.ofSeconds(3));
+  }
+
+  private void simulateUserActivity(Duration duration) {
+    simulateUserActivity(duration, Duration.ofMillis(500), 100);
+  }
+
+  private void simulateUserActivity(Duration duration, Duration waitDuration, int scrollDistance) {
+    while (!duration.isNegative()) {
+      jsActions.scrollBy(0, scrollDistance);
+      wait.forX(waitDuration);
+      jsActions.scrollBy(0, -scrollDistance);
+      duration = duration.minus(waitDuration);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Introduce a new variable instead of reusing the parameter "duration". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1226)